tendrl/commons/flows/create_cluster/__init__.py

```diff
@@ -42,6 +42,9 @@ def run(self):
                 "Space char not allowed in cluster name"
             )
 
+        # Execute super run() to execute pre-runs
```

Comment:
add to comment that "atoms and post_runs will also be executed due to this super call"

also add a "TODO (team) break down run() into run_pre(), run_atom(), run_post() where we run the pre_runs, atoms, post_runs respectively so run() simply calls run_pre(), run_atom(), run_post()